### PR TITLE
Update dependencies, cleanup irrelevant template comments, remove .idea from the project

### PR DESCRIPTION
**Pull request by Spikhalskiy**

### CODE REVIEW

File: src/main/java/moneytransferapp/AccountActivity.java [hunk 1]
```diff
@@ -3,12 +3,15 @@
 // @@@SNIPSTART money-transfer-project-template-java-activity-interface
 
 import io.temporal.activity.ActivityInterface;
+import io.temporal.activity.ActivityMethod;
 
 @ActivityInterface
 public interface AccountActivity {
 
+    @ActivityMethod
```
Comment (tsurdilo) [inline src/main/java/moneytransferapp/AccountActivity.java:11]: not needed. is it added for users? we dont use this annotation in samples either (sample for the other one)

Comment (Spikhalskiy) [reply src/main/java/moneytransferapp/AccountActivity.java:11]: Not needed, but I think that it's actually a good idea to use explicitly. Just from the code style and self-documentation perspective.

**Comment by tsurdilo** [comment]
@Spikhalskiy while we updating, in main readme (bottom) should we not switch the text to have user start the worker before the workflow?

File: src/main/java/moneytransferapp/AccountActivity.java (hunk 1 above)
Comment (Spikhalskiy) [reply src/main/java/moneytransferapp/AccountActivity.java:11]: But I will remove it if it's against our common practice.

**Comment by Spikhalskiy** [comment]
@tsurdilo but the whole point of the tutorial is that we start the workflow before the worker.

**Comment by Spikhalskiy** [comment]
@tsurdilo that's why I really thing that we should add a section where we explore the UI AFTER the worker made it's job.

File: src/main/java/moneytransferapp/AccountActivity.java (hunk 1 above)
Comment (tsurdilo) [reply src/main/java/moneytransferapp/AccountActivity.java:11]: It's fine :)

**Comment by tsurdilo** [comment]
@Spikhalskiy oops  sorry :) proceed then . +1

**Pull request merged**